frontend/src/components/common/Header.tsx

```diff
@@ -100,9 +118,52 @@ const Header = (): JSX.Element => {
               )}
               {authenticatedUser.role === Role.ADMIN && (
                 <>
-                  <Link as={ReactLink} to={Routes.VIEW_DONATIONS}>
-                    View Donations
-                  </Link>
+                  <Menu>
+                    <MenuButton>
+                      Donation & Shift Management{" "}
+                      {isNavDropdownOpen ? (
+                        <ChevronUpIcon />
+                      ) : (
+                        <ChevronDownIcon />
+                      )}
+                    </MenuButton>
+                    <MenuList>
+                      <MenuItem
+                        style={{
+                          borderRadius: "0.4rem",
+                          margin: "auto",
+                          width: "95%",
+                        }}
+                        _focus={{
+                          bg: "raddish.50",
+                        }}
+                        _active={{
+                          bg: "raddish.50",
+                        }}
+                      >
+                        <Link as={ReactLink} to={Routes.ADMIN_CHECK_INS}>
+                          Fridge Check-ins
+                        </Link>
```

Comment:
small detail and you might want to check with @stacykwok but the whole menu should be clickable rather than just the linked text right?
You can probably just remove the `Link` element and wrap the menu item with a clickable div or something else to route 👍🏻 
https://user-images.githubusercontent.com/19617248/158285011-b8ce7df1-bd81-45e2-86c0-760b6701973c.mov